src/engine/test/source/hlp/hlp_test.cpp

```diff
@@ -120,44 +122,135 @@ TEST(hlpTests, url_parsing)
     ASSERT_EQ("user", result["url.username"]);
 }
 
-TEST(json_test, dummy_test)
+// Test: parsing IP addresses
+TEST(hlpTests_IPaddress, IPaddress_success)
```

Comment:
We should refactor and rename this test and the below one to only test IPV4 formats. As IPV6 has its own tests.

Reply:
done